app/uk/gov/hmrc/thirdpartyapplication/controllers/binders/package.scala

```diff
@@ -66,4 +66,32 @@ package object binders {
       userId.value.toString()
     }
   }
+
+  implicit def developerIdentifierBinder(implicit textBinder: PathBindable[String]): PathBindable[DeveloperIdentifier] = new PathBindable[DeveloperIdentifier] {
```

Comment:
Looks OK but I'm not familiar with package objects, nor what the PathBindable is for in this context - so I'll assume it works but if in doubt don't rely solely on my review of this particular piece.